usaspending_api/api_docs/markdown/endpoints.md

```diff
@@ -104,6 +104,7 @@ The currently available endpoints are listed in the following table.
 |[/api/v2/search/spending_by_category/awarding_subagency/](/api/v2/search/spending_by_category/awarding_subagency/)|POST| Returns data that is grouped in preset units to support the Spending by Awarding Subgency data visualizations on USAspending.gov's Advanced Search page|
 |[/api/v2/search/spending_by_category/funding_agency/](/api/v2/search/spending_by_category/funding_agency/)|POST| Returns data that is grouped in preset units to support the Spending by Funding Agency data visualizations on USAspending.gov's Advanced Search page|
 |[/api/v2/search/spending_by_category/funding_subagency/](/api/v2/search/spending_by_category/funding_subagency/)|POST| Returns data that is grouped in preset units to support the Spending by Funding Subgency data visualizations on USAspending.gov's Advanced Search page|
+|[/api/v2/search/spending_by_category/country/](/api/v2/search/spending_by_category/country/)|POST| Returns data that is grouped in preset units to support the Spending by Country data visualizations on USAspending.gov's Advanced Search page|
```

Comment:
IIRC these should be in alphabetical order

Reply:
Sure thing, fixed.